### ISSUE TITLE
The 'navigation' object hasn't been initialized yet. This might happen if you don't have a navigator mounted, or if the navigator hasn't finished mounting. See https://reactnavigation.org/docs/navigating-without-navigation-prop#handling-initialization for more details.

### ISSUE THREAD

**Issue by chococascante** [nandorojo/solito#155]
Hi,
I get this error whenever I start the android simulator, and it pops up again when I try to use solito router to push a route. It works perfectly on web tho.

This is my NavigationContainer:
`import { NavigationContainer } from '@react-navigation/native'
import * as Linking from 'expo-linking'
import { useMemo } from 'react'

export function NavigationProvider({
  children,
}: {
  children: React.ReactNode
}) {
  return (
    <NavigationContainer
      linking={useMemo(
        () => ({
          prefixes: [Linking.createURL('/')],
          config: {
            initialRouteName: 'app',
            screens: {
              app: 'app',
              login: 'login',
              signup: 'signup',
            },
          },
        }),
        []
      )}
    >
      {children}
    </NavigationContainer>
  )
}
`

My provider:
`import { Dripsy, paperTheme } from './dripsy'
import { NavigationProvider } from './navigation'
import { FirebaseContextProvider } from 'app/contexts/use-firebase'
import { FirebaseAuthContextProvider } from 'app/contexts/use-firebase-auth-context'
import { Provider as PaperProvider } from 'react-native-paper'
import React from 'react'

export function Provider({ children }: { children: React.ReactNode }) {
  return (
    <NavigationProvider>
      <FirebaseContextProvider>
        <FirebaseAuthContextProvider>
          <PaperProvider theme={paperTheme}>
            <Dripsy>{children}</Dripsy>
          </PaperProvider>
        </FirebaseAuthContextProvider>
      </FirebaseContextProvider>
    </NavigationProvider>
  )
}
`

**Comment by nandorojo** [comment]
Unfortunately this example is too hard to parse. Also, your provider is empty, so that must be why it's broken. I need a better reproduction to help here, so I'm going to close for now. 

Have you referenced the starter monorepo?
